codes/c/chapter_array_and_linkedlist/list.c

```diff
@@ -0,0 +1,169 @@
+/**
+ * File: list.c
+ * Created Time: 2022-01-12
+ * Author: Zero (glj0@outlook.com)
+ */
+
+#include "../include/include.h"
+#include <assert.h>
```

Comment:
Keep *.h to include.h.

Reply:
> Keep *.h in include.h.

I understand, I don't have a PR for this part of the code, how did it come out?

It should be my mistake, I haven't finished writing yet